frontend/src/components/MapView/Legends/LegendItem/index.tsx

```diff
@@ -208,10 +209,29 @@ const LegendItem = memo(
       }
       return (
         <Grid item>
-          <Typography variant="h5">{children}</Typography>
+          {typeof children === 'string' ? (
+            <Markdown
+              linkTarget="_blank"
+              components={{
+                p: ({ children: pChildren }: { children: React.ReactNode }) => (
+                  <Typography
+                    variant="h5"
+                    className={classes.legendTextMarkdown}
+                  >
+                    {pChildren}
+                  </Typography>
+                ),
+              }}
+              allowedElements={['p', 'h5', 'strong', 'em', 'a']}
```

Comment:
Is there a reason to limit elements here? If so, it might be useful to reference allowed elements somewhere else. In the readme maybe or next to the the legendText type?

Reply:
Many of the theoretically allowable elements would not have acceptable formatting here (thinking of examples like tables, lists, code, etc). Rather than try to build reasonably acceptable formatting for all of these elements, I decided to limit to the elements I believe this feature request is intended for. 

Good idea, I'll add a note to the readme what exactly is supported